@kiva/kv-components/tests/unit/specs/utils/mapUtils.spec.js

```diff
@@ -1,6 +1,7 @@
 import {
 	getCoordinatesBetween,
-} from '../../../../utils/mapAnimation';
+	getLoansIntervals,
+} from '../../../../utils/mapUtils';
 
 describe('mapAnimation', () => {
```

Comment:
This block should be renamed to `mapUtils`, and the new describe below removed, the `describe('getLoansIntervals', () => {` moved inside this `describe`.